app/scripts/controllers/preferences.js

```diff
@@ -42,6 +42,7 @@ export default class PreferencesController {
       // set to false will be using the static list from contract-metadata
       useTokenDetection: false,
       useNftDetection: false,
+      useCurrencyRateCheck: false,
```

Comment:
I think we want to default this to `true`. Current behaviour in prod today is the same as if this was true